JakinChan200/CS180_Roald
# Created Express API

## Overview
Added a server folder for the backend code. Created a basic web server with Express supporting GET method for index and GET and POST for /api/ endpoint.


## Materials
- Issue #1 

## Pictures
Not applicable.

## Manual Testing Guide
- Make sure you have the repo cloned to your machine.
- Navigate to /server folder and run "npm install" to install dependencies and dev dependencies. (I might've messsed up and left node_modules/ in the commit, so this might not be necessary)
- In the /server folder, run "npm run app" as a command to start the server. Should see console output "Server running on port 8000".
- Now you can navigate to http://localhost:8000 and see "Hello World!" or to http://localhost:8000/api and see "GET route on api".
- Alternatively, use cURL with 'curl -X GET "http://localhost:8000"'
- Check the API route with 'curl -X GET "http://localhost:8000/api"' or 'curl -X POST "http://localhost:8000/api"'

Pull requests should receive approval before merge.
